### PR TITLE
Use alt+return to open package on github

### PR DESCRIPTION
**Pull request by KELiON**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 ## Usage
 Search for NPM packages using "**npm** [search term]".
 
-<kbd>Return</kbd> - Open a package on the NPM website.
-
-<kbd>cmd+c</kbd> - Copy the install command eg. `npm install -S react`
+* <kbd>Return</kbd> - Open a package on the NPM website;
```
Comment (JordanAdams) [inline README.md]: Can we remove the ";" at the end of this line and use "." instead 🙂

File: README.md [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
+* <kbd>alt+Return</kbd> - Open a package github repository.
```
Comment (JordanAdams) [inline README.md]: Let's lowercase "Return" here

File: src/index.js [hunk 3]
```diff
@@ -22,7 +22,13 @@ const displayResult = ({ display, actions }, result) => {
     title: result.package.name,
     subtitle: result.package.description,
     clipboard: `npm i -S ${result.package.name}`,
-    onSelect: () => actions.open(result.package.links.npm)
+    onSelect: (event) => {
+      const { npm, repository } = result.package.links
+      const url = event.altKey && repository
+        ? `${repository}#readme`
+        : npm
+      actions.open(url)
```
Comment (JordanAdams) [inline src/index.js]: We're missing semicolons in this block 👍

File: src/index.js [hunk 4]
```diff
@@ -22,7 +22,13 @@ const displayResult = ({ display, actions }, result) => {
     title: result.package.name,
     subtitle: result.package.description,
     clipboard: `npm i -S ${result.package.name}`,
-    onSelect: () => actions.open(result.package.links.npm)
+    onSelect: (event) => {
+      const { npm, repository } = result.package.links;
+      const url = event.altKey && repository
+        ? `${repository}#readme`
```
Comment (JordanAdams) [inline src/index.js]: Having just tried this I feel like I'd prefer this to not jump to the readme. The way I'm seeing this is that NPM already shows the readme and people would likely want the repo for things like PRs and issues.

What are your thoughts on this?

😃

Comment (KELiON) [reply src/index.js]: @JordanAdams agree, makes sense!

**Comment by JordanAdams** [comment]
LGTM

**Pull request merged**